contextualized/regression/metamodels.py

```diff
@@ -7,12 +7,12 @@
 from contextualized.modules import ENCODERS, Explainer, SoftSelect
 from contextualized.functions import LINK_FUNCTIONS
 
+METAMODELS = ["simple", "subtype", "multitask", "tasksplit"]
```

Comment:
Probably make this a dict like the other file summaries

Reply:
I posted a comment to issue 154 wondering about this, I noticed that `METAMODELS` and `MODELS` combined each seem to define every type of model (eg: "simple, univariate", "subtype, multivariate"). I wasn't sure if this was meant to be a grid-like feature or if you think it would just be better to keep these as dictionaries.

Reply:
I'm thinking a single-level dictionary with the full set of model-metamodel combos would be most useful now.